audit-ci.json

```diff
@@ -2,5 +2,5 @@
   "high": true,
   "pass-enoaudit": true,
   "retry-count": 20,
-  "allowlist": ["node-forge", "d3-color"]
+  "allowlist": ["d3-color", "knex", "json5", "jsonwebtoken"]
```

Comment:
Add json5 and knex to allowlist
jsonwebtoken is added to the list until @cumulus/api upgrade